core/shared/src/main/scala/fs2/Stream.scala

```diff
@@ -1361,7 +1362,7 @@ final class Stream[+F[_], +O] private (private val free: FreeC[Algebra[Nothing,
     * }}}
     */
   def handleErrorWith[F2[x] >: F[x], O2 >: O](h: Throwable => Stream[F2, O2]): Stream[F2, O2] =
-    Stream.fromFreeC(Algebra.scope(get[F2, O2]).handleErrorWith(e => h(e).get[F2, O2]))
+    Stream.fromFreeC(get[F2, O2].handleErrorWith(e => h(e).get[F2, O2]))
```

Comment:
No need for a scope insertion here anymore.

Reply:
cool. I remember this was done on the interruption, when we signalled it with the Throwable. If all interruption test go green this is ok.